addons/core/XEH_PREP.sqf

```diff
@@ -177,6 +179,8 @@ PREP_SUB(hint,showRadioVolume);
 PREP(setChannelFrequency);
 PREP(setRadioOwner);
 PREP(setLrSpeakers);
+PREP(setSwHalfDuplexOverride);
+PREP(setLrHalfDuplexOverride);
```

Comment:
Be careful about https://github.com/michail-nikolaev/task-force-arma-3-radio/pull/1269#discussion_r125027361 in the get Variant.